CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Missing dependencies in outter `package.json`.
```

Comment:
what did you mean by outter? `outer` maybe?

Reply:
Opa, good catch.